jha/src/components/enrolment/ReviewTableList.vue

```diff
@@ -1072,7 +1063,24 @@ export default {
     getFilePlural(count) {
       //takes an integer, returns single or plural form of word
       return (count === 1) ? 'file' : 'files';
-    }
+    },
+    getFormattedGender(genderInitial) {
+      if (!genderInitial) {
+        return "";
+      }
+      let result = "";
+      switch (genderInitial) {
+        case "F":
+          result = "Female";
+          break;
+        case "M":
+          result = "Male";
+          break;
+        default:
+          result = "Another Gender (X)";
+      }
+      return result;
```

Comment:
Instead of having a switch/case here, can you use the labels found in the radio options? So something like:
```javascript
getFormattedGender(gender) {
  const genderOption = radioOptions.find((item) => item.value === gender);
  return genderOption && genderOption.label ? genderOption.label : '';
}
```

Reply:
That way if we change the labels ever, then we won't have to update the review page as well.